### PR TITLE
Basic v1 price functions

### PR DESCRIPTION
**Pull request by Jeadie**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -308,7 +308,7 @@
 
 "@grpc/grpc-js@^1.8.14":
   version "1.8.21"
-  resolved "https://registry.yarnpkg.com/@grpc/grpc-js/-/grpc-js-1.8.21.tgz#d282b122c71227859bf6c5866f4c40f4a2696513"
```
Comment (ewgenius) [inline yarn.lock]: I don't think we should switch to yarn registry

File: src/client.ts [hunk 2]
```diff
@@ -99,94 +99,45 @@ class SpiceClient {
...
     }
 
-    const resp = await this.fetchInternal(`/v0.1/prices/${pair}`);
+    const resp = await this.fetchInternal(`/v1/prices/latest?pair=` + pairs.join(","));
     if (!resp.ok) {
-      throw new Error(
-        `Failed to get latest price: ${resp.statusText} (${await resp.text()})`
-      );
+        throw new Error(
+            `Failed to get latest prices V1: ${resp.statusText} (${await resp.text()})`
```
Comment (ewgenius) [inline src/client.ts]: nit: default indentation is 2 spaces

**Review by ewgenius — APPROVED** [review]

**Comment by ewgenius** [comment]
I think we'll need version bump to 0.6 as well

File: yarn.lock (hunk 1 above)
Comment (phillipleblanc) [reply yarn.lock]: You mean switch from yarn registry? This PR is switching it to the NPM registry from the yarn registry.

**Review by phillipleblanc — APPROVED** [review]

File: yarn.lock (hunk 1 above)
Comment (Jeadie) [reply yarn.lock]: Was accident. Will revert `yarn.lock` changes.

File: src/client.ts (hunk 2 above)
Comment (Jeadie) [reply src/client.ts]: reverted.

**Comment by Jeadie** [comment]
> I think we'll need version bump to 0.6 as well

Plan is to merge this, automate tagging/versioning and then launch v1.0

**Pull request merged**